src/Arcus.Observability.Telemetry.Core/Arcus.Observability.Telemetry.Core.csproj

```diff
@@ -19,16 +19,8 @@
     <RootNamespace>Arcus.Observability.Telemetry.Core</RootNamespace>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
-    <FrameworkReference Include="Microsoft.AspNetCore.App" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.1'">
-    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
-  </ItemGroup>
-
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.8" />
```

Comment:
If it was added with a condition here (https://github.com/arcus-azure/arcus.observability/pull/166/files#diff-94bb8bf830bf69c261860dcf4886e0c762d193d7bb8eda88b9bd7451618adce5L26), why are we always adding it now? Is it on purpose?

Reply:
Not sure what you're getting at? Is it about the packages or the separate HTTP project?

Reply:
We had a condition for this one first and now we always add it. I'm just wondering if that was on purpose or not; that's all

Reply:
It was on purpose. Apparently, the 3.1.8 version supports netstandard, but I'm still not sure if that means that we support netcoreapp2.2.

Reply:
We can drop 2.2 support if we have to

Reply:
I'd assume that .NET Core 2.2 also implements .NET Standard 2.